remote/src/main/resources/reference.conf

```diff
@@ -568,6 +568,11 @@ pekko {
         # set to 0b for platform default
         receive-buffer-size = 256000b
 
+        # Flush operations are generally speaking expensive, consolidating flushes will improve throughput
+        # but with the downside of a bit higher latency.
+        # The number of flushes after which an explicit flush will be done, set to 0 to turn this feature off.
+        explicit-flush-after-flushes = 0
```

Comment:
Yes, but if there is a reading in progress, then the flush will be done after the readComplete, otherwise, a flush task is scheduled to the event loop.

@laosijikaichele Would you like to take a look at this?

Reply:
@pjfanning I did not find a better name :(

Reply:
would we better off going through the code and making the write and flush calls in the right places - as per https://github.com/netty/netty/discussions/13608#discussioncomment-6983583

Reply:
@He-Pin I wish I can help, but I am really not familiar with Scala...

Reply:
@laosijikaichele Thanks, it's java too, here I want to add `FlushConsolidationHandler` to reduce the flushtimes, you can check the second commit.

Btw, I saw you did many performance optimization in Netty, you may want take Pekko/Akka a try too, where performance is essential, you will find much fun.

Reply:
@He-Pin , I still prefer to Java coding..., maybe someday in the future. Anyway, thanks for the invitation, appreciate it.:)

Reply:
> @He-Pin , I still prefer to Java coding..., maybe someday in the future. Anyway, thanks for the invitation, appreciate it.:)

Benefits from your improvements in Netty, thanks

Reply:
I would rename this conf to `flushes-after-explicit-flush`

Reply:
> I would rename this conf to `flushes-after-explicit-flush`

Ha, both ok, I just use the `explicitFlushAfterFlushes`,which means will only do explicit flush after N times flush.
How  about `explict-flush-after`?

Reply:
@He-Pin I think adding an extra config is a worse solution than fixing our code so that it calls flush in the right places.

Please read https://github.com/netty/netty/discussions/13608#discussioncomment-6983583

Reply:
@pjfanning The problem's what the default parameter should be `3` or `5`?

Reply:
This is the text from the link

```
 Ideally you want to be explicit about when you write and when you flush
```

Please read https://github.com/netty/netty/discussions/13608#discussioncomment-6983583

Reply:
@pjfanning There is only one `write` method from the `AssociationHandle`, In netty 4, the `wirte` method will not do automatic flusing, so the real behavior alian the netty 3 version is call `writeAnfFlush`.
But that will introduce much flush/system call.

If we don't want to add any default configuration, we may:
1. do nothing
2. add a `FlushConsolidationHandler` with `FlushConsolidationHandler(true)`.

Reply:
And the `Ideally ` means more improvemnt, in our internal rpc implementation, we send all write to a writeQeueu and do explicitly flush based on `size/event/time`.

Reply:
Could we try `add a FlushConsolidationHandler with FlushConsolidationHandler(true)` and see if it works well?

Simpler than expecting our users to tune this for themselves.

Reply:
The the default value is `256`, so you suggest to we go this first, ok, that's a plan too.